scanpy/preprocessing/pca_for_sparse.py

```diff
@@ -0,0 +1,70 @@
+
+import numpy as np
+from distutils.version import StrictVersion
```

Comment:
`distutils.version` is broken. Please use `from packaging import version` and then `version.parse`.